### PR TITLE
Update travis job

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
**Issue**
Update the travis yaml to complete the build setup in libres instead of using python file from ert.

### CODE REVIEW

File: .libecl_version [hunk 1]
```diff
@@ -0,0 +1 @@
+export LIBECL_VERSION=2.3.2
```
Comment (markusdregi) [inline .libecl_version]: I think you want this one: 2.4.b1

Comment (lars-petter-hauge) [reply .libecl_version]: I think so too! :P

File: .travis.yml [hunk 2]
```diff
@@ -60,40 +63,51 @@ addons:
...
-    - hash -r
-    - conda config --set always_yes yes --set changeps1 no
-    - conda update -q conda
-    # Useful for debugging any issues with conda
-    - conda info -a
-    - wget https://raw.githubusercontent.com/Statoil/ert/master/travis/install_python_packages.py
-    - python install_python_packages.py
+  - if [[ "$CC" == "gcc" ]]; then export CXX="g++-4.8"; fi
+  - cmake --version
+  - if [[ $PYTHON_VERSION == 2.7 ]]; then
```
Comment (markusdregi) [inline .travis.yml]: I think you can remove all of the `TRAVIS_PYTHON_VERSION` :)

Comment (lars-petter-hauge) [reply .travis.yml]: Yeppsi! Fixed :)

**Review by markusdregi — APPROVED** [review]
Nicely executed, good git history!

**Pull request merged**